rsc/extra/deploy_doc.sh

```diff
@@ -8,7 +8,9 @@
 # files. Travis will automatically commit the changes
 # and push it to the repo
 
+# Legacy ODOC directory hidden in a ODOC_DIR folder
 VERSION=$1
+OLD_ODOC_DIR := "odoc"
```

Comment:
As far as I can tell from the PR, the sphinx doc is complementary to the odoc documentation: the sphinx doc is more of a cli-user facing doc, whereas the API doc is a dev and lib-user facing doc. In particular, I didn't see any thing that would make sphinx generate the API doc (but maybe I'm wrong).
Assuming I'm right, I find the use of `OLD` and `Legacy` wording particularly misleading considering and actually wrong.